js/src/scatter.js

```diff
@@ -50,7 +64,10 @@ var ScatterView = widgets.WidgetView.extend( {
             arrow: this.geo_arrow,
             sphere: this.geo_sphere,
             cat: this.geo_cat,
+            point: this.geo_point,
+            tri: this.geo_tri
         }
+        this.sprite_geos = ['point', 'tri'];
```

Comment:
what about prefixing the names?
sprite_point or 2d_point

Reply:
I like 2d_point -- will change this.